bridge/irc/irc.go

```diff
@@ -242,15 +242,33 @@ func (b *Birc) Send(msg config.Message) (string, error) {
 	return "", nil
 }
 
+func colorizeString(s string, foreground int64, background int64) string {
+	return fmt.Sprintf("\x03%02d,%02d%s\x03", foreground, background, s)
+}
+
+func (b *Birc) colorizeUsername(username string) string {
+	ircNickColors := b.GetStringSlice2D("ColorNicksList")
+	checksum := int(crc32.ChecksumIEEE([]byte(username)))
+	selectedColorPair := ircNickColors[checksum%len(ircNickColors)]
+	foreground, err := strconv.ParseInt(selectedColorPair[0], 16, 64)
+	if err == nil {
+		panic(fmt.Sprintf("Bad IRC color code: %v", selectedColorPair[0]))
```

Comment:
Panics are not allowed for this, you'll have to handle this gracefully

Reply:
What the possible ways of handling this error gracefully?

Reply:
Maybe do a `b.Log.Errorf("Bad IRC color code: %v.", selectedColorPair[0])` and use a fixed foreground/background color pair to pass